simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -268,9 +268,10 @@ void doRequest(CollectorRegistry registry, String job, Map<String, String> group
         writer.close();
       }
 
-      int response = connection.getResponseCode();
-      if (response != HttpURLConnection.HTTP_ACCEPTED) {
-        throw new IOException("Response code from " + url + " was " + response);
+      int responseCode = connection.getResponseCode();
+      if (responseCode != HttpURLConnection.HTTP_ACCEPTED) {
+        throw new IOException("Response code from " + url + " was " + responseCode
+            + ", message: " + connection.getResponseMessage());
```

Comment:
In my example setup, line 273 is reached:

`java.io.IOException: Response code from http://localhost:9091/metrics/job/logmatic_sift was 500, message: Internal Server Error`

However, the body of that response contains, which is never exposed on the client side:

`text format parsing error in line 28: second HELP line for metric name "total_read"`

This is the recommended way of sending error messages in HTTP.  Surely there is a simple way of accessing that message in Java?